src/ray/protobuf/common.proto

```diff
@@ -128,6 +136,24 @@ message TaskSpec {
   int32 max_retries = 16;
 }
 
+
+message Bundle {
+  bytes bundle_id = 1;
+  map<string, double> unit_resources = 2;
+  int64 unit_count = 3;
```

Comment:
Do we need unit count in the first iteration? Consider leaving it out and implementing it later if needed.